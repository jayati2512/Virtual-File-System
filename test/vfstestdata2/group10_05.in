createvfs demo 1024
mountvfs demo
makedir /home/ medir
makedir /home/medir youdir
makedir /home/medir/youdir itdir
listdir /home/medir 1 listoutput
unmountvfs demo
