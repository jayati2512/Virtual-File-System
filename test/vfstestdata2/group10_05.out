createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
listdir_SUCCESS
unmountvfs_SUCCESS
